packages/react/src/components/Reaction/Reaction.tsx

```diff
@@ -126,6 +126,6 @@ Reaction.slotClassNames = {
 }
 
 /**
- * A reaction is used to indicate user's reaction.
+ * A Reaction indicates user's emotion or perception.
```

Comment:
Should identify more clearly why this is different from button?

Reply:
actually, there is no much difference between these two components. From those things that we may suggest in the description those are
- `Button`, by means of its API, provides set of props that better describe button's semantics
- `Reaction` is more suitable for the chat message cases, and it has far less scenarios for customization supported

However, main reason behind `Reaction` component was to introduce button-like element that would have separate styles in theme, because of its different semantic meaning. If there would be a better mechanism to add this variant of `Button`, it would be chosen then, without need for a dedicated component.

Will introduce sentence that suggests close relation of this component with `Chat.Message`